cwms-data-api/src/main/java/cwms/cda/api/StreamController.java

```diff
@@ -160,13 +155,7 @@ public void getOne(@NotNull Context ctx, @NotNull String streamId) {
         try (Timer.Context ignored = markAndTime(GET_ONE)) {
             DSLContext dsl = getDslContext(ctx);
             StreamDao dao = new StreamDao(dsl);
-            String unitSystem = queryParamAsClass(ctx, new String[]{UNIT_SYSTEM, },
-                    String.class, UnitSystem.SI.getValue(), metrics,
-                    name(StreamController.class.getName(), GET_ONE));
-            String stationUnits = "km";
-            if(UnitSystem.EN.getValue().equals(unitSystem)) {
-                stationUnits = "mi";
-            }
+            String stationUnits = ctx.queryParam(STATION_UNIT) == null ? "mi" : ctx.queryParam(STATION_UNIT);
```

Comment:
if you do queryParamAsClass and pass in String.class you can call getOrDefault for the "mi". Slightly more readable than a ternary.

Reply:
updated to use queryParamAsClass rather than a ternary